governance/default.go

```diff
@@ -863,6 +863,16 @@ func (gov *Governance) removeDuplicatedVote(vote *GovernanceVote, number uint64)
 	gov.RemoveVote(vote.Key, vote.Value, number)
 }
 
+// There can be special case when governance changed but blockNumber gets smaller. (ex. block rewinding)
+// Then we cannot use UpdateCurrentSet as usual.
+// Instead, CreateCurrentSet is used in this case.
+func (gov *Governance) CreateCurrentSet(num uint64) {
+	newNumber, newGovernanceSet, _ := gov.ReadGovernance(num)
+	gov.actualGovernanceBlock.Store(newNumber)
+	gov.currentSet.Import(newGovernanceSet)
+	gov.triggerChange(newGovernanceSet)
+}
+
```

Comment:
Can we first update the actualGovernanceBlock and then call UpdateCurrentSet()?
This seems redundant.